service/src/test/java/datawave/microservice/audit/health/rabbit/RabbitHealthCheckerTest.java

```diff
@@ -82,13 +82,13 @@ public void everythingHealthyTest() throws Exception {
         
         healthChecker.runHealthCheck();
         
-        Assert.assertTrue(healthChecker.isHealthy());
-        Assert.assertEquals(Status.UP, healthChecker.health().getStatus());
+        Assertions.assertTrue(healthChecker.isHealthy());
```

Comment:
consider updating this class to follow the static import convention for junit assertion methods

Reply:
same comment as above.